tests/test_app.py

```diff
@@ -62,6 +62,27 @@ def test_metrics(app, now, enrollment):
 
 
 @responses.activate
+def test_metrics_no_usage(app, now, enrollment):
+    empty_month_data = """"Usage Data Extract",
```

Comment:
adressed in https://github.com/blue-yonder/azure-cost-mon/pull/10/commits/67c42396aad41a856e7bee69dacb12087e03121d